File: src/main/java/pl/longhorn/rebETL/model/comment/Comment.java
package pl.longhorn.rebETL.model.comment;

import lombok.*;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Lob;
import java.time.LocalDate;

@Entity
@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor(access = AccessLevel.PRIVATE)
public class Comment {
    @Id
    private Integer id;
    private String nick;
    private LocalDate date;
    private Integer productRating;
    private Integer commentRating;
    @Lob
    private String text;
}

File: WebScraperAngular/src/app/shared/comment.ts
export interface Comment {
    id:number;
    commentRating:number;
    date:Date;
    nick:string;
    productRating?:number;
    text:string;
    url:string;
}
export class Comment {
    id:number;
    commentRating:number;
    date:Date;
    nick:string;
    productRating?:number;
    text:string;
    url:string;
}